scripts/skills/ccb.sh

```
@@ -3,4 +3,9 @@ set -euo pipefail
 MODE="--interactive"
 if [ -n "${BENCHSIGHT_AUTO:-}" ]; then MODE="--auto"; fi
 cd "$(dirname "$0")/../.."
-PYTHONPATH=. ./.venv/bin/python3 -m src.benchsight.cli ccb "$@" $MODE
+source "$(dirname "$0")/_resolve_python.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./_resolve_python.sh was not specified as input (see shellcheck -x).

(SC1091)